### DISCUSSION TITLE
file about starting with '_'

### DISCUSSION THREAD

**Discussion by DrAugus** [community/community#19794] [Projects and Issues] [answered]
## error

Can't get this file `_commonjsHelpers.b8add541.js`

![image](https://user-images.githubusercontent.com/34206897/176808905-d5935bf0-e648-4489-98c3-d7d1489be6c6.png)

## [repo](https://github.com/DrAugus/draugus.github.io) 

![image](https://user-images.githubusercontent.com/34206897/176808928-f9be047e-30de-4828-8fdb-947c5b88a05a.png)

## url 

- [error page1](https://draugus.github.io/life/food)
- [error page2](https://draugus.github.io/life/game/genshin/wish)

**Comment by huabin** [comment]
你这里的文件目录被自动转义了

**Reply by DrAugus** [reply]
文件名原因还是什么？其他页面莫得问题欸，求教。

**Reply by huabin** [reply]
你尝试去掉 js 文件前的下划线试试，将该文件改名称不带 `_` 试试。

**Reply by DrAugus** [reply]
我也想去掉😂我不知道这个文件咋生成出来的

**Reply by huabin** [reply]
网站是用什么生成的？

**Reply by DrAugus** [reply]
vuepress

**Comment by icdance** [comment, marked as answer]
默认以 `_`, `.`, `#` 开始，或者以`~`结束的文件名会被忽略。可以通过编辑`_config.yml`文件来修改 include 参数来设置允许的文件。

具体参考：

By default, Jekyll doesn't build files or folders that:

- are located in a folder called /node_modules or /vendor
- start with _, ., or #
- end with ~
- are excluded by the exclude setting in your configuration file
- If you want Jekyll to process any of these files, you can use the include setting in your configuration file.

https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll

**Reply by icdance** [reply]
还有一种方法是在repo的根目录下新建 文件名`.nojekyll`的文件，这样可以bypass所有Jekyll的规则。

**Reply by DrAugus** [reply]
非常感谢，用了`.nojekyll`好使了！
